Userland/Libraries/LibWeb/Communication/CloseEvent.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/DOM/Event.h>
+
+namespace Web::Communication {
+
+class CloseEvent : public DOM::Event {
+public:
+    using WrapperType = Bindings::CloseEventWrapper;
+
+    static NonnullRefPtr<CloseEvent> create(const FlyString& event_name, bool was_clean, u16 code, const String& reason)
+    {
+        return adopt_ref(*new CloseEvent(event_name, was_clean, code, reason));
+    }
+
+    virtual ~CloseEvent() override { }
```

Comment:
I know historically serenity has been using empty destructors like this but recently I see (and use) more and more `= default` - presumably because clang-tidy in CLion complains about it :D

```suggestion
    virtual ~CloseEvent() override = default;
```